packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -50,6 +55,10 @@ typedef ComputeImpl = Future<R> Function<Q, R>(ComputeCallback<Q, R> callback, Q
 /// method should be used to execute parallel tasks that reduce the risk of
 /// skipping frames.
 ///
+/// [compute] internally uses [Isolate.run] and provides additional timing
+/// information. If you don't want or need the additional timing information,
+/// you might be interested in using [Isolate.run] directly.
+///
```

Comment:
We should just have a line under See also about `Isolate.run`. We should not document implementation details here. If anything, we could say that this method works similarly to `Isolate.run` with additional tracing.

Reply:
Would just adding:

```dart
/// See also:
/// ...
/// * [Isolate.run], for an alternative version of this function without tracing information
```

be fine?